hdfs_namenode/tox.ini

```diff
@@ -16,5 +18,7 @@ passenv =
     COMPOSE*
 commands =
     pip install -r requirements.in
-    unit: pytest -m"not integration" -v
-    integration: pytest -m"integration" -v
+    unit: pytest -m"unit" -v
+    integration: pytest -v
```

Comment:
Isn't integration running all tests now? Is that what we want?

Reply:
It is not running everything due to https://github.com/DataDog/integrations-core/pull/4187/files#diff-ed4fe80493829a3f0ad88aa927a3a7e4R24. It's the trick found by @therve https://github.com/DataDog/integrations-core/pull/4180/files#diff-2045763de1fd605870c5ca10ead8854eR25